vajram/extensions/fory/vajram-fory-codegen/README.md

```
@@ -6,13 +6,13 @@
 
 This module plugs into the Krystal codegen pipeline via the
 `ModelsCodeGeneratorProvider` SPI.  When the annotation processor detects a
-`@ModelRoot` annotated with `@SupportedModelProtocols(Fory.class)`, this
+`@ModelRoot` annotated with `@SupportedModelProtocol(Fory.class)`, this
 generator produces an `_ImmutFory` wrapper class for that model.
 
 ## Architecture
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)